README.md

```diff
@@ -56,6 +56,14 @@ heroku config:set DOMAIN_ADMIN_USERNAME=x@gooddata.com DOMAIN_ADMIN_PASSWORD=xy
 git push heroku HEAD:master
 heroku open
 ```
+### Enable Geo Chart for Storybook and Example
```

Comment:
Could you please change this?
`Example` => `Live Examples`

Reply:
fixed